src/lib.rs

```diff
@@ -94,15 +94,20 @@ fn take_first_chunk<'a, const N: usize>(slice: &mut &'a [u8]) -> Option<&'a [u8;
 
 impl FxHasher {
     /// Creates `fx` hasher with a given seed.
-    pub fn with_seed(seed: usize) -> FxHasher {
+    pub const fn with_seed(seed: usize) -> FxHasher {
         FxHasher { hash: seed }
     }
+
+    /// Create a default `fq` hasher.
```

Comment:
Typo: `fq`

Reply:
Also: what's the point of creating a `default` function in this `impl` when `Default` is already impl'd? I'm actually surprised this even works.

Reply:
The point is that it can be const. I think a different name would be nicer though.

Reply:
`Default` is not a `#[const_trait]` yet and feature `const_trait_impl`/`effects` is still very much WIP. The inherent associated function `default` can be used in const context contrary to the trait associated function. Inherent associated items have precedence over their trait counterpart.

Reply:
Hm, it could be called `new`